rocAL/rocAL/source/pipeline/tensor.cpp

```diff
@@ -85,17 +90,31 @@ bool operator==(const rocalTensorInfo &rhs, const rocalTensorInfo &lhs) {
             rhs.layout() == lhs.layout());
 }
 
-void rocalTensorInfo::reallocate_tensor_roi_buffers() {
-    _roi = std::make_shared<std::vector<RocalROI>>(_batch_size);
+void rocalTensorInfo::allocate_tensor_roi_buffers() {
+    size_t roi_size = _batch_size * 4 * sizeof(unsigned);
+    if(_mem_type == RocalMemType::HIP) {
+#if ENABLE_HIP
```

Comment:
add the condition for OPENCL

Reply:
Since there is no OPENCL support for augmentations in RPP we have not added the allocation for OpenCL ROI buffers